.editorconfig

```diff
@@ -23,9 +24,14 @@ indent_size = 2
 [webapp/i18n/**.json]
```

Comment:
This can also be removed.

Reply:
Oh yeah, I forgot about that when I was copying the file to the plugin in the first place. I'll remove it